swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -29,16 +32,28 @@ enum LogLevel {
     case assert, error, warning, info, debug
 }
 
+/// For the hardcoded values that represent the privacy settings we have an example of more enum utilization here.
```

Comment:
This is where you might reference back to the KB:
- https://help.fullstory.com/hc/en-us/articles/360044349073-FullStory-Private-by-Default#exclude-fs-exclude
- https://help.fullstory.com/hc/en-us/articles/360044349073-FullStory-Private-by-Default#mask-fs-mask
- https://help.fullstory.com/hc/en-us/articles/360044349073-FullStory-Private-by-Default#unmask-fs-unmask

I'd focus on what your API is contributing and less about the language itself ("an example of more enum utilization").